docker/.env.example

```
@@ -14,6 +14,18 @@ POSTGRES_PORT=5432
 REDIS_HOST=redis
 REDIS_PORT=6379
 
+# Observability
+GRAFANA_PORT=3000
+GRAFANA_USER=admin
+GRAFANA_PASSWORD=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAFANA_PASSWORD key should go before the GRAFANA_PORT key

(UnorderedKey)